index.html

```diff
@@ -0,0 +1,275 @@
+        <section class="works">
+            <div class="snapshot-3"></div>
+         
+            <div class="info">
+               <div class="title">Multi-Post Stories</div> 
+                <div class="frame">
+                    <div class="client">
+                        <div class="name">
+                            CANOPY
+                        </div>
+                    </div>
+                    <div class="counter"></div>
+    
+                    <div class="role">
+                        <div class="text">Back End Dev</div>
+                    </div>
+                    <div class="counter"></div>
+    
+                    <div class="role">
+                        <div class="text">2015</div>
+                    </div>
+                </div>
+    
+                <div class="desc">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+    
+                <div class="tags">
+                    <div>
+                        <li class="tag"><a href="">html</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">CSS</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">Javascript</a></li>
+                    </div>
+                </div>
+    
+                <div class="action">
+                    <div class="btn-action">
+                        <a class="see">See Project</a>
+                    </div>
+                </div>
+            </div>
+    
+    
+        </section>
+        <section class="works">
+            <div class="snapshot-4"></div>
+         
+            <div class="info">
+               <div class="title">Tonic</div> 
+                <div class="frame">
+                    <div class="client">
+                        <div class="name">
+                            CANOPY
+                        </div>
+                    </div>
+                    <div class="counter"></div>
+    
+                    <div class="role">
+                        <div class="text">Back End Dev</div>
+                    </div>
+                    <div class="counter"></div>
+    
+                    <div class="role">
+                        <div class="text">2015</div>
+                    </div>
+                </div>
+    
+                <div class="desc">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </div>
+    
+                <div class="tags">
+                    <div>
+                        <li class="tag"><a href="">html</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">CSS</a></li>
+                    </div>
+                    <div>
+                        <li class="tag"><a href="">Javascript</a></li>
+                    </div>
```

Comment:
Kindly use li tags in a nav tag, The nav tag is for semantics reasons and the right position for convention and user experience for mobile devices.